register.py

```diff
@@ -97,8 +102,12 @@ def main():
     # TODO: temporary fix for https://github.com/HTPhenotyping/registration/issues/17
     if htcondor.param["AUTH_SSL_CLIENT_CAFILE"] == "/etc/ssl/certs/ca-bundle.crt":
         htcondor.param["AUTH_SSL_CLIENT_CAFILE"] = "/etc/ssl/certs/ca-certificates.crt"
-
-    success = request_token(pool=args.pool, resource=args.host)
+    
+    if args.local_dir:
+        # user specified path for their local directory
+        success = request_token(pool=args.pool, resource=args.host, local_dir=args.local_dir)
+    else:
+        success = request_token(pool=args.pool, resource=args.host)
```

Comment:
```suggestion
    success = request_token(pool=args.pool, resource=args.host, local_dir=args.local_dir)
```

`args.host` will always be set since we gave it a default of `None` so the if/else is superfluous

Reply:
My bad. I thought it would raise a KeyError, but if that's the case accessing local_dir in if statement would also raise that, which contradicts here.